web/src/pages/Dashboard/Dashboard.scss

```diff
@@ -392,7 +396,7 @@ footer {
 }
 
 .btn:hover {
-    background: #0d8b28b4;
+    background: #9067c5b4;
```

Comment:
same here, since you use the same value twice, it might be better to do it that way